src/styles/StudentStyles/index.less

```diff
@@ -12,16 +13,16 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 0% 10%;
+    padding: 0% 2%;
+    margin: 0% 5%;
+    width: 25%;
 }
 
 .studentdb-right-column {
     display: flex;
     flex-direction: column;
     justify-content: center;
-    align-items: center;
-    padding: 0% 10%;
-    
+    width: 100%;    
```

Comment:
Does the width need to be set to 100% because of inheritance? That is, if you remove this line, the width will not be 100%? As far as I understand, width is 100% by default.